test/mocha/mock-credential.json

```diff
@@ -0,0 +1,62 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    {
+      "ex": "https://example.org/examples#",
+      "schema": "http://schema.org/",
+      "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+
+      "3rdPartyCorrelation": "ex:3rdPartyCorrelation",
+      "AllVerifiers": "ex:AllVerifiers",
+      "Archival": "ex:Archival",
+      "BachelorDegree": "ex:BachelorDegree",
+      "Child": "ex:Child",
+      "CLCredentialDefinition2019": "ex:CLCredentialDefinition2019",
+      "CLSignature2019": "ex:CLSignature2019",
+      "IssuerPolicy": "ex:IssuerPolicy",
+      "HolderPolicy": "ex:HolderPolicy",
+      "Mother": "ex:Mother",
+      "RelationshipCredential": "ex:RelationshipCredential",
+      "UniversityDegreeCredential": "ex:UniversityDegreeCredential",
+      "ZkpExampleSchema2018": "ex:ZkpExampleSchema2018",
+
+      "issuerData": "ex:issuerData",
+      "attributes": "ex:attributes",
+      "signature": "ex:signature",
+      "signatureCorrectnessProof": "ex:signatureCorrectnessProof",
+      "primaryProof": "ex:primaryProof",
+      "nonRevocationProof": "ex:nonRevocationProof",
+
+      "alumniOf": {"@id": "schema:alumniOf", "@type": "rdf:HTML"},
+      "child": {"@id": "ex:child", "@type": "@id"},
+      "degree": "ex:degree",
+      "degreeType": "ex:degreeType",
+      "degreeSchool": "ex:degreeSchool",
+      "college": "ex:college",
+      "name": {"@id": "schema:name", "@type": "rdf:HTML"},
+      "givenName": "schema:givenName",
+      "familyName": "schema:familyName",
+      "parent": {"@id": "ex:parent", "@type": "@id"},
+      "referenceId": "ex:referenceId",
+      "documentPresence": "ex:documentPresence",
+      "evidenceDocument": "ex:evidenceDocument",
+      "spouse": "schema:spouse",
+      "subjectPresence": "ex:subjectPresence",
+      "verifier": {"@id": "ex:verifier", "@type": "@id"}
+    }
```

Comment:
Almost all of this is unnecessary for the VC being produced by the test... can we remove the terms that are not used?

Reply:
There's a FIXME related to this.  What are the requirements about context support?  Currently the `https://www.w3.org/2018/credentials/examples/v1` is not part of our bedrock document loader framework.